package.json

```diff
@@ -357,6 +357,7 @@
     "postcss": "^8.3.6",
     "postcss-loader": "4.3.0",
     "postinstall-postinstall": "^2.1.0",
+    "prettier": "^2.4.1",
```

Comment:
Why is this added?

Reply:
To be able to use `matchInlineSnapshot()`

Reply:
It is a peer dependency then?

Reply:
We need prettier in dependencies because we are using jest v26. v27 doesn't need that. See https://github.com/facebook/jest/pull/11566
And https://github.com/facebook/jest/issues/8467